meorg_client/client.py

```diff
@@ -139,12 +142,12 @@ def logout(self):
         return response
 
 
-    def get_file_status(self, file_id: str) -> Union[dict, requests.Response]:
+    def get_file_status(self, id: str) -> Union[dict, requests.Response]:
```

Comment:
All of these ids had to be explicitly called "id" for compatibility with automated curation of the mocker fixture.

Reply:
Also more consistent with the openapi spec from ME.org.